nntrainer/layers/layer_node.cpp

```diff
@@ -256,11 +220,12 @@ std::ostream &operator<<(std::ostream &out, const LayerNode &l) {
 }
 
 std::string LayerNode::getDistLayerType() const {
-  if (getDistribute())
-    return std::static_pointer_cast<TimeDistLayer>(layerv1)->getDistLayerType();
-  else
-    throw std::runtime_error(
-      "Get distribution layer type for non-distributed layer");
+  // if (getDistribute())
```

Comment:
Intended comment?

Reply:
Yes, this is intended. This will be enabled + updated in the next patch where the distribution layer is enabled for V2 design.